### DISCUSSION TITLE
Issues on running Geoclimate for the first time - sun.reflect

### DISCUSSION THREAD

**Discussion by lucianaswf** [orbisgis/geoclimate#874] [Q&A]
Hi, 

I am trying to use Geoclimate for the first time and I have no knowledge of coding.

I followed the steps at: https://github.com/orbisgis/geoclimate/wiki/Default-case-with-OSM-for-Windows. However, I got the message: 

WARNING: sun.reflect.Reflection.getCallerClass is not supported. This will impact performance. Please set at least one OSM filter. e.g osm : ['A place name']
Cannot execute the OSM workflow.

I did not change anything in the configuration file.

Can someone help me? Do you know what this issue is about and how can I solve it?

**Comment by j3r3m1** [comment]
Hi @lucianaswf thank you for your message.
I suppose you have used the 0.0.1 version while the wiki is valid for the 0.0.2 SNAPSHOT version. The configuration file was different for version 0.0.1.
We recommend using the 0.0.2 version downloadable here: https://nightly.link/orbisgis/geoclimate/workflows/CI_snapshot/master/geoclimate-0.0.2-SNAPSHOT.zip

**Reply by lucianaswf** [reply]
Thank you. It worked!

Now, I have to run it for Sao Paulo, with my local database. I would really appreciate if anyone has a tutorial for that.
Best,
Luciana

**Reply by ebocher** [reply]
Tutorial is here https://github.com/orbisgis/geoclimate/wiki/Default-case-with-OSM-for-Linux

**Reply by lucianaswf** [reply]
São Paulo does not have OSM for the entire city. But I have a very complete database. Is it possible to run Geoclimate with my data?

**Reply by ebocher** [reply]
Please be carefull about the Sao Paulo area https://nominatim.openstreetmap.org/ui/search.html?q=sao+paulo
It's very big. It will be nice to test on a small BBox. See http://bboxfinder.com/#-23.563122,-46.655588,-23.530547,-46.612072 for an example.
You can set [-23.563122,-46.655588,-23.530547,-46.612072] as location input

**Reply by lucianaswf** [reply]
Ok. I will do that. 
However there are some parts of the city that do not have data in OSM. 
What is your recommendation in this case?

**Reply by j3r3m1** [reply]
Using your own data is then a solution. But then it is a bit more complicated to run since you need to create your own script using groovy language. We should have simple scripts to use for this purpose soon but they are not finished yet. Are you using GeoClimate for research purpose ? Or study ? Or application ?

**Reply by lucianaswf** [reply]
I will use it for research purpose. I am a postdoc at the University of São Paulo and I am doing a research stay at the University of Toulouse (until April 2024). 
GeoClimate data will be used to run TempMap (https://www.sciencedirect.com/science/article/pii/S2212095518303523) and maybe other climate analysis. Also, the São Paulo Cityhall is interested in some metrics for their analysis,
Is there any planned deadline for those simpler scripts? Or, would you be interested in collaborate with this project?

**Reply by j3r3m1** [reply]
OK nice to hear. It will be before the end of April 2024, hopefully before the end of 2023 but not 100% sure. Come back to us at the beginning of January if fine for you ?

**Reply by lucianaswf** [reply]
Beginning of January would be a little tight due to the sequence of tasks I will have to do... but yes, I can contact you in January.
Hope I can use Geoclimate for São Paulo!!!

**Reply by lucianaswf** [reply]
Hi @j3r3m1 
Any uptades on this topic?

**Reply by j3r3m1** [reply]
Hi @lucianaswf unfortunately not yet available. I will let you know once it is available, might be mid-February. Sorry for inconvenience

**Comment by j3r3m1** [comment]
@ebocher if we do not have a release soon, I suggest we recommend using the 0.0.2 in the wiki ?

**Reply by j3r3m1** [reply]
Changed made

**Comment by ebocher** [comment]
There is a message for the documentation "The documentation is inline with the current GeoClimate code source."
But sure we must recommend to use the current snaphot.
